### PR TITLE
Fix issue #645

### PR DESCRIPTION
**Pull request by leosoaivan**
* Divide contents into two rows
* Mobile: Add margin-top to success stories content
* Mobile: Add centering and display:block to student-name elements
* Mobile: Add centering to photo
* Flush stories content to photo
* Vertically align name/title to photo

### CODE REVIEW

**Comment by leosoaivan** [comment]
@arun1595

File: config/database.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ development:
   adapter: postgresql
   database: theodinproject
   host: localhost
+  username:
```
Comment (KevinMulhern) [inline config/database.yml]: Can we remove this file from the PR please, `git checkout origin/master config/database.yml` should do it

File: app/assets/stylesheets/components/success_stories.scss [hunk 2]
```diff
@@ -12,15 +12,36 @@
...
     }
 
+    .info {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    .student-name {
+      // margin-bottom: 10px;
```
Comment (KevinMulhern) [inline app/assets/stylesheets/components/success_stories.scss]: can we remove this commented out code please?

File: app/assets/stylesheets/components/success_stories.scss [hunk 3]
```diff
@@ -12,15 +12,36 @@
...
+
+      @media(max-width: 600px) {
+      // @include media-breakpoint-down(xs) {
```
Comment (KevinMulhern) [inline app/assets/stylesheets/components/success_stories.scss]: Can we remove this commented out code as well please?

**Comment by KevinMulhern** [comment]
This is looking really good @leosoaivan, just left a couple of minor comments.

**Comment by KevinMulhern** [comment]
Thanks for making those changes @leosoaivan 
LGTM 👍

File: db/seeds/dummy_projects.rb [hunk 4]
```diff
@@ -48,7 +48,6 @@ def users
     user = User.new(user_details)
     user.password = "password"
     user.password_confirmation = "password"
-    user.skip_confirmation!
```
Comment (arun1595) [inline db/seeds/dummy_projects.rb]: Why was this removed?

We could also add `user.confirm` to make sure the user account is confirmed.

Comment (KevinMulhern) [inline db/seeds/dummy_projects.rb]: We aren't requiring email confirmations anymore Arun. I turned off that feature.
